package.json

```diff
@@ -1,128 +1,90 @@
-      "/test/"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 90,
-        "functions": 95,
-        "lines": 95,
-        "statements": 95
-      }
-    },
-    "collectCoverageFrom": [
-      "src/*.{js,ts}"
-    ]
-  },
-  "prettier": {
-    "semi": true,
-    "singleQuote": true
+    "semantic-release-prepare": "ts-node tools/semantic-release-prepare"
   },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional"
-    ]
+  "private": true,
+  "dependencies": {
+    "@types/resize-observer-browser": "^0.1.3",
+    "document-register-element": "1.13.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "^8.3.5",
-    "@commitlint/config-conventional": "^8.3.4",
-    "@types/cordova-plugin-network-information": "^1.3.2",
-    "@types/jest": "^25.1.3",
-    "@types/node": "^13.7.7",
-    "colors": "^1.4.0",
-    "commitizen": "^4.0.3",
-    "coveralls": "^3.0.9",
-    "cross-env": "^7.0.1",
-    "cz-conventional-changelog": "^3.1.0",
+    "@nrwl/eslint-plugin-nx": "9.0.4",
+    "@nrwl/jest": "^9.0.4",
+    "@nrwl/node": "^9.0.4",
+    "@nrwl/web": "^9.0.4",
+    "@nrwl/workspace": "9.0.4",
+    "@types/jest": "25.1.4",
+    "@types/node": "^13.9.0",
+    "@typescript-eslint/eslint-plugin": "^2.22.0",
+    "@typescript-eslint/parser": "^2.22.0",
+    "core-js": "^3.6.4",
+    "dotenv": "8.2.0",
+    "eslint": "6.8.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-prettier": "^3.1.2",
     "husky": "^4.2.3",
-    "jest": "^25.1.0",
-    "jest-config": "^25.1.0",
+    "jest": "25.1.0",
+    "jest-environment-puppeteer": "^4.4.0",
     "lint-staged": "^10.0.8",
-    "lodash.camelcase": "^4.3.0",
     "prettier": "^1.19.1",
-    "prompt": "^1.0.0",
-    "replace-in-file": "^5.0.2",
-    "rimraf": "^3.0.2",
-    "rollup": "^1.32.0",
-    "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-json": "^4.0.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-sourcemaps": "^0.5.0",
-    "rollup-plugin-typescript2": "^0.26.0",
-    "rxjs": "^6.5.4",
+    "puppeteer": "^2.1.1",
     "semantic-release": "^17.0.4",
-    "shelljs": "^0.8.3",
     "travis-deploy-once": "^5.0.11",
-    "ts-jest": "^25.2.1",
-    "ts-node": "^8.6.2",
-    "tslint": "^6.0.0",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-config-standard": "^9.0.0",
-    "typedoc": "^0.16.11",
-    "typescript": "3.8.3"
+    "ts-jest": "25.2.1",
+    "ts-node": "~8.6.2",
+    "tslint": "~6.0.0",
+    "typescript": "~3.8.3",
+    "cypress": "^3.8.2",
+    "@nrwl/cypress": "9.0.4"
   },
-  "peerDependencies": {
-    "rxjs": ">=6.0.0"
+  "lint-staged": {
+    "{src}/**/*.ts": [
+      "prettier --write",
+      "git add"
+    ]
   },
-  "dependencies": {
-    "@types/resize-observer-browser": "^0.1.3"
-  }
+  "files": [
+    "dist"
+  ],
+  "author": "Jan-Niklas Wortmann <6104311+JWO719@users.noreply.github.com>",
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "contributors": [
+    {
```

Comment:
I really appreciate everything you implemented so far, but there is more needed than submitting one pr to be listet as contributor. Let's see how things go on, but for now I'd appreciate if you could remove this.